WordPress/twentytwentyone
# Clean up custom properties: Remove unused, update use where appropriate

Fixes #486, after the revert of #540

## Summary

I wrote up a quick (hacky) script to run through the CSS files & pull out unused variables, and this PR removes those. It also updates some other variables:

- Update and use the `--branding--color-link-hover` variable
- Remove `--heading--line-height` since it was just a clone of `--global--line-height-heading`, and replaces the use of it in the CSS with the appropriate header level line-height
- I also brought in the relative font-size updates from #540
- Remove `--primary-nav--justify-content` since the value doesn't change anything in the menu display

## Test instructions

This PR can be tested by following these steps:

1. View the site, and make sure there are no visual changes
2. Specifically around headings & navigation (where the most variables were changed/removed)

## Quality assurance
